src/detect.h

```diff
@@ -451,6 +451,29 @@ typedef struct DetectEngineAppInspectionEngine_ {
     struct DetectEngineAppInspectionEngine_ *next;
 } DetectEngineAppInspectionEngine;
 
+typedef struct DetectContentData_ {
```

Comment:
would like to keep this in detect-content.h, can this be reworked to make that happen?